ci-operator/step-registry/stackrox/install-helm/stackrox-install-helm-commands.sh

```diff
@@ -215,11 +219,19 @@ wait_deploy central
 get_init_bundle
 install_secured_cluster_with_helm
 echo ">>> Wait for 'stackrox-secured-cluster-services' deployments"
+if [[ "${ROX_SCANNER_V4:-true}" == "true" ]]; then
+  wait_deploy scanner-v4-indexer
+  wait_deploy scanner-v4-matcher
+  wait_deploy scanner-v4-db
+fi
 wait_deploy scanner
 wait_deploy scanner-db
 wait_deploy sensor
 wait_deploy admission-control
 
+echo "Restart sensor to accelerate scanner start up..."
+oc rollout restart deployment sensor -n stackrox
+
```

Comment:
How did you come to that conclusion? How come the Scanner initialization has this dependency on Sensor?